apps/governance/.env

```diff
@@ -32,3 +32,4 @@ LC_ALL="en_US.UTF-8"
 # Cosmic elevator flags
 NX_SUCCESSOR_MARKETS=true
 NX_METAMASK_SNAPS=true
+NX_PRODUCT_PERPETUALS=false
```

Comment:
Question: should these feature flags also be in the e2e env files or are they on by default?

Reply:
This feature only makes extra API requests when the flag is explicitly set to true, so nothing will fall over if it's not set. I can set the flag for the e2e env files in this PR if you'd like, but equally it's fine to wait until you pick up when you write the tests